examples/gnrc_border_router/Makefile.esp.conf

```diff
@@ -0,0 +1,5 @@
+# Set the SSID and password of your WiFi network here
+CFLAGS += -DESP_WIFI_SSID=\"Your_WiFi_name\"
+CFLAGS += -DESP_WIFI_PASS=\"Your_secure_password\"
+
+FEATURES_REQUIRED += esp_wifi
```

Comment:
You already include `esp_wifi` as a module in `Makefile.board.dep` so requiring `esp_wifi` here is only enforcing blacklisting.